src/content/docs/new-relic-solutions/get-started/glossary.mdx

```diff
@@ -66,12 +66,40 @@ Whether you're considering New Relic or you're already using our capabilities, t
     id="agent"
     title="agent"
   >
-    A New Relic agent is an installable piece of software that integrates with multiple types of technologies (for example, web frameworks, operating systems, and types of databases) and reports data to New Relic, usually on a specific cadence. We have <InlinePopover type="apm" /> agents, an infrastructure agent, a <InlinePopover type="browser" /> agent, and mobile monitoring agents. The various technologies that our agents integrate with are just some of [our many integrations](https://newrelic.com/instant-observability).
+    A New Relic agent is an installable piece of software that integrates with multiple types of technologies (for example, web frameworks, operating systems, and types of databases) and reports data to New Relic, usually on a specific cadence. All agents are integrations, but not all integrations are agents. Many of our agents support multiple integrations; for example, the Java agent collects data from a variety of Java frameworks and technologies, or the infrastructure agent enables the on-host Cassandra integration.
+
+Here's a list of what we consider New Relic agents:
+
+* APM:
+	* C SDK agent
+	* Go agent
+	* Java agent
+	* .NET agent
+	* Node.js agent
+	* PHP agent
+	* Python agent
+	* Ruby agent
+* Infrastructure:
+	* Infrastructure agent
+	* Linux agent
```

Comment:
@ally-sassman could you confirm that we have agents for Linux, MacOS, Unix, and Windows? I had thought that we just had the Unix server integration (which covers a variety of Nixes) and a Windows integration (not agent). I'm not really clear on the difference between what those agents record and what's recorded by the infrastructure agent that can run in those nix systems and Windows.